integration/extensions-proto3/parameters.txt

```diff
@@ -0,0 +1 @@
+outputExtensions=true,unknownFields=true,outputServices=generic-definitions,outputServices=default
```

Comment:
I initially thought I could get this working with `outputExtensions=true`, but actually I suspect this needs to be an entirely new flag `outputCustomOptions=true` ?